test/cppsuite/test_harness/core/component.h

```diff
@@ -63,7 +60,8 @@ class component {
         debug_print("Loading component: " + _name, DEBUG_INFO);
         _enabled = _config->get_optional_bool(ENABLED, true);
         _throttle = throttle(_config->get_optional_int(OPS_PER_SECOND, 1));
-        _running = true;
+        /* If we're not enabled we should've be running. */
+        _running = _enabled && true;
```

Comment:
I am not sure I am getting this. Isnt _running = enabled enough ?

Reply:
Yes......